frontend/src/Button.css

```diff
@@ -0,0 +1,7 @@
+.Button {
```

Comment:
so this is a stylistic thing I've been doing in react css. I like to create a (s)css file to go 1 for 1 with a component file. I like to capitalize the class name to mirror how React components are namespaced.